src/local-ssh/ipc/extensionServiceServer.ts

```diff
@@ -108,23 +109,28 @@ class ExtensionServiceImpl implements ExtensionServiceImplementation {
             // TODO(lssh): Get auth info according to `request.gitpodHost`
             const gitpodHost = this.hostService.gitpodHost;
 
-            const rawWorkspace = await this.sessionService.getAPI().getWorkspace(actualWorkspaceId, _context.signal);
-            const wsData = rawWorkspaceToWorkspaceData(rawWorkspace);
+            const ws = await this.sessionService.getAPI().getWorkspace(actualWorkspaceId, _context.signal);
 
-            const ownerToken = await this.sessionService.getAPI().getOwnerToken(actualWorkspaceId, _context.signal);
+            instanceId = ws.status!.instance!.instanceId;
```

Comment:
```suggestion
            instanceId = ws.status?.instance?.instanceId ?? '';
```

Reply:
this one cannot be empty?

Reply:
I will mark it as a resolved, since i did not change this line and we did not see any issues with it.